include/gtsam_points/factors/integrated_vgicp_factor.hpp

```diff
@@ -69,6 +69,14 @@ class IntegratedVGICPFactor_ : public gtsam_points::IntegratedMatchingCostFactor
     return static_cast<double>(inliers) / correspondences.size();
   }
 
+  gtsam::NonlinearFactor::shared_ptr clone() const override {
```

Comment:
Can we simplify this method like that of `IntegratedGICPFactor`?

Reply:
done